scripts/drive-prs.md

```
@@ -0,0 +1,354 @@
+  --field pull-request-number="<NUMBER>" \
+  --field prompt="Please review this PR. Assess the changes for correctness, \
+code quality, and alignment with the project standards in DEVELOPING.md. \
+If the PR looks good, approve it. If changes are needed, request them."
+```
+
+Then **wait** for the review to be submitted before continuing.
+
+### Step 6a — Address review feedback
+
+1. Read the review comments:
+   ```bash
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)